### ISSUE TITLE
instalation guide please..

### ISSUE THREAD

**Issue by damai21** [jeag2002/ProgrammaticEngineMigration#1]
Hi man, can you teach me to install this script on server??

**Comment by jeag2002** [comment]
Yes of course, which script do you want to install (this project contains
tons of mini projects)?...what are your setup? (work with
Linux?/Windows?...Eclipse?/Intelj IDEA?)

**Comment by damai21** [comment]
I currently learn RTB (OpenSSP) https://github.com/ad-tech-group/openssp that script use Java Language, i've successfull Openssp but i don't understand to use that.. like how to integration other endpoint and how to generate tags or request ads..

I want to learn to install 
RTBRestProjectReactiveServer => Spring Web Flux Rest Reactive Server

I use Linux Ubuntu 16

Please teach me.. thanks for reply..
